users/views.py

```diff
@@ -19,6 +25,42 @@ class UserTokenObtainPairView(TokenObtainPairView):  # type: ignore
     serializer_class = UserTokenObtainPairSerializer
 
 
+class UserFollowView(APIView):
+    def get_object(self, pk: Any) -> Any:
+        try:
+            return User.objects.get(pk=pk)
+        except User.DoesNotExist:
+            raise Http404
+
+    def get(self, request: Any, pk: Any, format: Any = None) -> Any:
+        user = self.get_object(pk)
+        serializer = UserFollowingSerializer(user)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+
+    def post(self, request: Any, pk: Any, format: Any = None) -> Any:
+        user = request.user
+        follow = self.get_object(pk)
+        serializer = CreateFollowingSerializer(
+            data={"follower": user.id, "followed": follow.id}
+        )
+        if serializer.is_valid(raise_exception=True):
```

Comment:
No need for the if statement just have it as:
serializer.is_valid(raise_exceptions=True)
serializer.save()
Then the response